pages/common/git-show-index.md

```diff
@@ -0,0 +1,12 @@
+# git show-index
+
+> Show packed archive index.
+> More information: <https://git-scm.com/docs/git-show-index>.
+
+- Read the '.idx' file for a Git packfile and dump it's contents (stdout):
```

Comment:
Dumb typo (but I also have been working for 8 hours now so... meh =]

Reply:
It's really not that dumb...even native speakers like can make that mistake because there are odd nuances regarding 'its', and 8 hours is quite a long time to be at your desk!